web/pdf_page_view.js

```diff
@@ -508,33 +509,22 @@ class PDFPageView {
     this._textHighlighter.enable();
   }
 
-  /**
-   * @private
-   */
-  _resetZoomLayer(removeFromDOM = false) {
-    if (!this.zoomLayer) {
+  #resetCanvas() {
+    const { canvas } = this;
+    if (!canvas) {
       return;
     }
-    const zoomLayerCanvas = this.zoomLayer.firstChild;
-    this.#viewportMap.delete(zoomLayerCanvas);
-    // Zeroing the width and height causes Firefox to release graphics
-    // resources immediately, which can greatly reduce memory consumption.
-    zoomLayerCanvas.width = 0;
-    zoomLayerCanvas.height = 0;
-
-    if (removeFromDOM) {
-      // Note: `ChildNode.remove` doesn't throw if the parent node is undefined.
-      this.zoomLayer.remove();
-    }
-    this.zoomLayer = null;
+    canvas.remove();
```

Comment:
Shouldn't the `canvas` be removed from the `#viewportMap` as well?

Reply:
The `#viewportMap` is a `WeakMap` so we don't care, right ?